certified-connectors/ZohoSign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1661 @@
+                      "description": "Folder name"
+                    },
+                    "request_name": {
+                      "type": "string",
+                      "description": "Request Name"
+                    },
+                    "modified_time": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "modified time"
+                    },
+                    "is_deleted": {
+                      "type": "boolean",
+                      "description": "Status of deletion"
+                    },
+                    "expiration_days": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "Expiration days"
+                    },
+                    "in_process": {
+                      "type": "boolean",
+                      "description": "In process"
+                    },
+                    "is_sequential": {
+                      "type": "boolean",
+                      "description": "Sequential status"
+                    },
+                    "request_type_name": {
+                      "type": "string",
+                      "description": "Request type name"
+                    },
+                    "owner_first_name": {
+                      "type": "string",
+                      "description": "Owner first name"
+                    },
+                    "folder_id": {
+                      "type": "string",
+                      "description": "Folder id"
+                    },
+                    "request_id": {
+                      "type": "string",
+                      "description": "Request id"
+                    },
+                    "request_type_id": {
+                      "type": "string",
+                      "description": "Request type id"
+                    },
+                    "owner_last_name": {
+                      "type": "string",
+                      "description": "Owner Last name"
+                    },
+                    "sign_percentage": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "Sign percentage"
+                    }
+                  },
+                  "description": "requests"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "Message"
+                },
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                }
+              }
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              },
+              "charset": {
+                "description": "charset",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
```

Comment:
@ZohoSign-integ Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**